From b27d14b589e05ee185acf79e6ab4b16a9dd51015 Mon Sep 17 00:00:00 2001
From: Ylarod <me@ylarod.cn>
Date: Thu, 20 Jul 2023 10:01:20 +0800
Subject: [PATCH 08/10] Florida: pool-frida

---
 src/frida-glue.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/frida-glue.c b/src/frida-glue.c
index 6154034e..dcbd4211 100644
--- a/src/frida-glue.c
+++ b/src/frida-glue.c
@@ -56,6 +56,8 @@ frida_init_with_runtime (FridaRuntime rt)
     g_io_module_openssl_register ();
 #endif
 
+    g_set_prgname ("ggbond");
+
     if (runtime == FRIDA_RUNTIME_OTHER)
     {
       main_context = g_main_context_ref (g_main_context_default ());
-- 
2.43.0.windows.1

